cmd/sidecar-injector/main.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
```

Comment:
While Cisco may be ok with you assigning Cisco copyright, you may want to put your own employer here. In lieu of that, I'd put your own name here. Same comment for all copyrights here. :)

Reply:
done thanks.